proposals/lip-0045.md

```diff
@@ -219,7 +219,7 @@ In this section, we specify the substores that are part of the Interoperability
 | `MAX_RESERVED_ERROR_STATUS` | uint64 | 63 | The largest error code reserved for the Interoperability module. |
 | `EMPTY_BYTES` | bytes | "" | The empty byte string. |
 | `EMPTY_FEE_ADDRESS` | bytes | "" | The empty byte string. |
-| `MIN_RETURN_FEE` | uint64 | 1000 | The minimum return fee for a cross-chain message in Beddows. |
+| `MIN_RETURN_FEE_PER_BYTE` | uint64 | 1000 | The minimum return fee per byte for a cross-chain message in Beddows. |
```

Comment:
1. Since it is the value for specific channels, we may also choose a more specific name like MIN_RETURN_FEE_PER_BYTE_LSK.
2. I think it would good to add here that this is the min return fee "for channels between the mainchain and a sidechain". Or even "for channels between the mainchain and a sidechain where the LSK token is used for message fees."

Reply:
changed the name and updated the description